module/x/gravity/migrations/v2/keys.go

```diff
@@ -0,0 +1,6 @@
+package v2
+
+const (
+	// ModuleName is the name of the module
+	ModuleName = "gravity"
```

Comment:
We don't need this file just to define the module name, it's not going to change between versions.